__tests__/linkdrop.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { Runner, toYocto, createKeyPair, BN, tGas } from "..";
+
+describe(`Running on ${Runner.getNetworkFromEnv()}`, () => {
+  jest.setTimeout(60000);
+  let runner: Runner;
+
+  beforeAll(async () => {
+    runner = await Runner.create(async ({ runtime }) => ({
+      linkdrop: await runtime.createAndDeploy(
+        "linkdrop",
+        `${__dirname}/build/debug/linkdrop.wasm`
+      ),
+    }));
+  });
+
+  test("call myself", async () => {
+    await runner.run(async ({ root, linkdrop }) => {
+      // Create temporary keys for access key on linkdrop
+      const senderKey = createKeyPair();
+      const public_key = senderKey.getPublicKey().toString();
+
+      // This adds the key as a function access key on `create_account_and_claim`
+      await root.call(
+        linkdrop,
+        "send",
+        {
+          public_key,
+        },
+        {
+          attachedDeposit: toYocto("2"),
+        }
+      );
+      // can only create subaccounts
+      const new_account_id = `bob.${linkdrop.accountId}`;
+      const actualKey = createKeyPair();
+      const new_public_key = actualKey.getPublicKey().toString();
+
+      let res = await linkdrop.call_raw(
+        linkdrop,
+        "create_account_and_claim",
+        {
+          new_account_id,
+          new_public_key,
+        },
+        {
+          signWithKey: senderKey,
+          gas: tGas("50"),
+        }
+      );
+      // @ts-ignore
+      console.log(
+        res.receipts_outcome
+          .map((o: any) => o.outcome.logs)
+          .filter((x: any) => x.length > 0)[0]
+      );
+    });
+  });
+
+  test("call myself", async () => {
```

Comment:
Do both tests get run? They have the same names.

Reply:
```suggestion
  test("Use `claim` to transfer to an existing account", async () => {
```